menu/server_configuration.sh

```diff
@@ -82,6 +80,16 @@ https://www.techandme.se/open-port-80-443/"
         rm -f "$SCRIPTS/activate-tls.sh"
```

Comment:
Is this removal needed? I would just remove the file, if tls was activated, no?

Reply:
It gets removed here:
https://github.com/nextcloud/vm/blob/4a6402ebfd7c5e642091643094988db216c40fa2/lets-encrypt/test-new-config.sh#L78
So I think the removal isn't necessary at this  place before the actual activation of tls

Reply:
It would ensure to always get the latest script, but at the same time it's actually not me who wrote it ;)

To me it doesn't matter really, we could do run_script instead, but then it would remove the logic in the startup script as we want all scripts to be pre-feteched to ensure a complete run.

Reply:
Aah, I remember now why they are there. Sometimes it bugs out and the scripts aren't removed. 

It's just there for cleaning purposes. Please leave it as is.

Reply:
Please see the discussion below. It is related